### PR TITLE
[MVC-959] Map Pop-Up Experience on Mobile

### PR DESCRIPTION
**Pull request by tiagovarela**
Changes to look out for (mobile)
1) Removed container frame and made the map full width
2) Heading and Key positioning
3)Pop-Up box for country information text display
4) Pop-Up box length

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -246,6 +246,62 @@ button {
...
+      margin-left: 0;
+    }
+}
+
+
+
+
+
+
+
```
Comment (raguiar9080) [inline _sass/_base.scss:304]: These empty lines are probably not needed ✂️

File: _sass/_map.scss [hunk 2]
```diff
@@ -194,9 +194,9 @@
     font-size: 14px;
     color: $textColorLight;
     margin: -5px 0 0;
-    max-width: 450px;
+    width: 320px;
 
-    table {
+  /*  table {
```
Comment (raguiar9080) [inline _sass/_map.scss:199]: Is this supposed to be removed since it's now a comment? If it is, probably fully removing is best.

**Comment by tiagovarela** [comment]
@SydneyMhashu I saw that you changed the `description` keys of a `.js` file. Can you explain those changes?

File: _includes/map.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div class="t-one {{include.colour}} {{include.classList}}" uk-scrollspy="target: > div; cls: uk-animation-fade; delay: 100">
-  <div class="wrapper">
+  <div class="map_Container">
```
Comment (raguiar9080) [inline _includes/map.html:2]: I would make this `map_container` to make it match the styling used throughout 👍

**Review by raguiar9080 — COMMENTED** [review]
Left a few comments but overall looking good 👍

File: _sass/_map.scss (hunk 2 above)
Comment (SydneyMhashu) [reply _sass/_map.scss:199]: Yes let me remove it. I just kept the code to be safe in case something else breaks

**Comment by SydneyMhashu** [comment]
@tiagovarela i took out descriptions because, description were already defined, and then there was some "description" text in the pop-up table that was causing the pop-up not break on mobile view.

**Comment by andregdias** [comment]
Can we have some screenshots of before/after?

**Comment by SydneyMhashu** [comment]
Screenshot 2021-01-22 at 16 59 13
Screenshot 2021-01-22 at 17 00 43
Screenshot 2021-01-22 at 17 01 31

**Pull request merged**